### PR TITLE
Add Cloud Profiler agent

### PR DESCRIPTION
**Pull request by ymotongpoo**
Add [Cloud Profiler](https://cloud.google.com/profiler) in the app.

Additionally, add `fmt.Errorf` to wrap the errors emit from underlying functions for better error handling.

### CODE REVIEW

**Comment by ymotongpoo** [comment]
This PR solves the profiler part in #2

File: main.go [hunk 1]
```diff
@@ -5,21 +5,35 @@ import (
...
+	"cloud.google.com/go/profiler"
 )
 
+const (
+	serviceName = "ogp-app"
+	serviceVersion = "1.0.0" // TODO: replace this
+	maxRetry = 3
+)
+
+func init() {
```
Comment (achiku) [inline main.go]: Do we need this?

Comment (ymotongpoo) [reply main.go]: Sorry I forgot to remove that part. The intention here was to check necessary directories before `main()` and avoid the error from `os.Open()`. I'll raise another issue just to tackle that.

**Comment by achiku** [comment]
Merged

**Pull request merged**